### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mtomko**
Adds a slightly richer example to the suite local fixture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +133,12 @@ class SuiteLocalExampleSuite extends CatsEffectSuite {
   test("second test") {
     IO(myFixture()).assertEquals(())
   }
```
Comment (danicheg) [inline README.md:135]: Let's remove this test, it doesn't bring any utility
```suggestion
```

**Review by danicheg — APPROVED** [review]
Although this example doesn't differ much from existing ones, but if we would slightly polish them overall, I'm for this refactoring 👍🏻

**Comment by mtomko** [comment]
Having now used this feature and written this example, I agree that the new example doesn't literally show anything that wasn't visible before, but I think having the allocated resource be something more concrete than `()` makes it a little easier to see what's going on.

File: README.md (hunk 1 above)
Comment (mtomko) [reply README.md:135]: Interesting, maybe I should work my example so we can show a test using a resource, and then using the same resource again. I think that's why there's a `"second test"` here.

**Pull request merged**